### PR TITLE
Fix log metrics bug 491

### PR DESCRIPTION
**Pull request by zhangjianweibj**
standard log level has seven types including:AUDIT,CRITICAL,DEBUG,INFO,TRACE,ERR,WARN.but monasca log metrics component only support warning and error types to transform to metrics(log.warn log.error).and user can not config that types.it is not a rational disign.

### CODE REVIEW

**Comment by zhangjianweibj** [comment]
ok

File: monasca-log-metrics/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ ENV CONFIG_TEMPLATE=true \
   KAFKA_URI=kafka:9092 \
   ZOOKEEPER_URI=zookeeper:2181 \
   KAFKA_WAIT_FOR_TOPICS=log-transformed,metrics
```
Comment (matrixik) [inline monasca-log-metrics/Dockerfile]: Missing `\` at the end. Without it Dockerfile is not working

File: monasca-log-metrics/README.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ Configuration
 | `KAFKA_WAIT_FOR_TOPICS`   | `log-transformed,metrics` | Topics to wait on at startup |
 | `KAFKA_WAIT_RETRIES`      | `24`             | # of kafka wait attempts           |
 | `KAFKA_WAIT_DELAY`        | `5`              | # seconds to wait between attempts |
+| `LOG_LEVEL`               | `warning,error,fatal` | Metric name need to transformed be from log  |
```
Comment (matrixik) [inline monasca-log-metrics/README.md]: Change description to: `List all log levels converted to metrics`

File: monasca-log-metrics/log-metrics.conf.j2 [hunk 3]
```diff
@@ -24,7 +24,7 @@ filter {
   }
 
   # drop logs with log level not in warning,error
```
Comment (matrixik) [inline monasca-log-metrics/log-metrics.conf.j2]: Replace `warning,error` with `LOG_LEVEL`

Comment (zhangjianweibj) [reply monasca-log-metrics/log-metrics.conf.j2]: ok,very thanks

**Pull request merged**